security-rules.bolt

```diff
@@ -137,12 +141,12 @@ path /landsraad/votingRights {
 
 path /landsraad/questions {
   read() { isSignedIn() }
-  write() { isSwiss() }
+  write() { isSignedIn() }
 }
```

Comment:
Also please change `database-sample-data.json` and possibly `database-constants.md` (it should be updated with every change od db structure)

Reply:
Ah ok! Will do.

Reply:
`database-sample-data.json` changed. I think there is nothing for `database-constants.md`.